app/controllers/parks_controller.rb

```diff
@@ -12,6 +12,10 @@ def edit
     @park = Park.find(params[:id])
   end
 
+  def create
+    redirect_to parks_path(params.park_id)
+  end
+
```

Comment:
I don't believe you should need this action in your non-API controller. Can you give me more information on why you included this?

Reply:
Dan told us that the API controller should only be responsible for sending and receiving data.  We get a double render error because we can't render json and redirect in the same controller os we had to render somewhere else.